docs/storefront/graphql/examples/index.mdx

```diff
@@ -208,4 +208,18 @@ query {
 }
 ```
 
-<a href="https://developer.bigcommerce.com/graphql?playground_tab=popularBrands" target="_blank">**Try it in GraphQL Playground**</a>
+<a href="https://developer.bigcommerce.com/graphql-storefront/playground" target="_blank">**Try it in GraphQL Playground**</a>
+
+## Get a favicon to Storefront GraphQL
+
+```graphql filename="Example query: Get a favicon" showLineNumbers copy
```

Comment:
```suggestion
```graphql filename="Example query: Get the storefront's favicon" showLineNumbers copy
```